### ISSUE TITLE
Create a Rust SDK

### ISSUE THREAD

**Issue by Skarlso** [gaia-pipeline/gaia#97]

**Comment by michelvocks** [comment]
gRPC currently does not officially support Rust but we could use this one for now: https://github.com/stepancheg/grpc-rust
What do you think?

**Comment by Skarlso** [comment]
Yep I know. I did a little digging. That thing isn't maintained either any longer. Might be enough though. I'll have to do a skeleton first.

**Comment by Skarlso** [comment]
Firs the reference of why I'm reluctant to do this as of yet:
https://github.com/grpc/grpc/issues/9316

I don't think they'll ever will incorporate it properly. And the one linked above is absolutely abandoned.

I could probably take it over, but my time is pretty thin stretched as is.

**Comment by michelvocks** [comment]
True. Let us down prfioritize this feature and if you got time and you just do it. 🤗

**Comment by Skarlso** [comment]
Started here https://github.com/gaia-pipeline/rustsdk

**Comment by Skarlso** [comment]
Uh, I really should pick this up... :D
